### PR TITLE
Update Usage messages to conform to new Usage message contributor guidelines

### PR DESCRIPTION
**Pull request by pajlada**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

### CODE REVIEW

**Comment by pajlada** [comment]
@Felanbird @leon-richardt can I have your views on this PR?

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -215,3 +215,29 @@ Keep the element on the stack if possible. If you need a pointer or have complex
...
+- Angle brackets are reserved for `<required arguments>`.
+- The word _Usage_ should be capitalized and must be followed by a colon.
+- If the usage deserves a description, put a dot after all parameters and explain it briefly.
+
+##### Good
+
+- `Usage: /block <user>`
+- `Usage: /unblock <user>. Unblocks a user.`
+- `Usage: /streamlink <channel>`
+- `Usage: /streamlink <channel>`
```
Comment (leon-richardt) [inline CONTRIBUTING.md:235]: These two examples are the same?

**Review by leon-richardt — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -215,3 +215,29 @@ Keep the element on the stack if possible. If you need a pointer or have complex
...
 - If you have to explicitly delete an object use `variable->deleteLater()` instead of `delete variable`. This ensures that it will be deleted on the correct thread.
 - If an object doesn't have a parent, consider using `std::unique_ptr<Type, DeleteLater>` with `DeleteLater` from "src/common/Common.hpp". This will call `deleteLater()` on the pointer once it goes out of scope, or the object is destroyed.
+
+## Conventions
+
+#### Usage strings
+
+When informing the user about how a command is supposed to be used, we aim to follow [this standard](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html) where possible.
+
+- Square brackets are reserved for `[optional options]`.
```
Comment (Felanbird) [inline CONTRIBUTING.md:225]:
```suggestion
- Square brackets are reserved for `[optional arguments]`.
```

**Review by Felanbird — CHANGES_REQUESTED** [review]

**Review by leon-richardt — APPROVED** [review]

**Review by Felanbird — APPROVED** [review]

**Review by pajlada — COMMENTED** [review]
image

**Pull request merged**